### PR TITLE
husky is used for git-hook - add dependency checker - fix dependency …

### PR DESCRIPTION
**Pull request by mommel**
…audit warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -277,10 +278,20 @@
     "redux-thunk": "^2.3.0",
     "source-map-support": "^0.5.9"
   },
+  "resolutions": {
+    "@octokit/rest":"16.33.1"
```
Comment (Illumiated) [inline package.json:282]: Will we get issues with that or is it only used by flow-typed

File: audit/audit-output.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...id":786,"path":"babel-jest>babel-plugin-istanbul>test-exclude>micromatch>braces","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"1.8.5","paths":["babel-jest>babel-plugin-istanbul>test-exclude>micromatch>braces"]}],"id":786,"created":"2019-02-15T21:44:30.680Z","updated":"2019-04-02T18:18:29.356Z","deleted":null,"title":"Regular Expression Denial of Service","found_by":{"link":"","name":"Santosh Rao"},"reported_by":{"link":"","name":"Santosh Rao"},"module_name":"braces","cves":[],"vulnerable_versions":"<2.3.1","patched_versions":">=2.3.1","overview":"Versions of `braces` prior to 2.3.1 are vulnerable to Regular Expression Denial of Service (ReDoS). Untrusted input may cause catastrophic backtracking while matching regular expressions. This can cause the application to be unresponsive leading to Denial of Service.","recommendation":"Upgrade to version 2.3.1 or higher.","references":"- [GitHub Commit](https://github.com/micromatch/braces/commit/abdafb0cae1e0c00f184abbadc692f4eaa98f451)","access":"public","severity":"low","cwe":"CWE-185","metadata":{"module_type":"","exploitability":4,"affected_components":""},"url":"https://npmjs.com/advisories/786"}}}
```
Comment (Illumiated) [inline audit/audit-output.json:1]: Can we add a  
rm audit-output.json as 2L to remove this file when no issues found

File: .yarnscripts/check-deps [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+DIRNAME=$(dirname "$0")
+if [ "${DIRNAME}" == ".yarnscripts" ]; then
```
Comment (Illumiated) [inline .yarnscripts/check-deps:3]: check lower

File: package.json [hunk 4]
```diff
@@ -277,10 +278,20 @@
...
     "node": ">=7.x",
     "npm": ">=4.x",
     "yarn": ">=0.21.3"
   },
-  "browserslist": "electron 1.6"
+  "browserslist": "electron 1.6",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn test",
```
Comment (Illumiated) [inline package.json]: dyt we can add check-deps to the pre-push

**Review by Illumiated — CHANGES_REQUESTED** [review]
Please check my comments, ty

File: audit/audit-output.json (hunk 2 above)
Comment (mommel) [reply audit/audit-output.json:1]: For sure, but i will only empty the file instead of a complete removement

File: package.json (hunk 1 above)
Comment (mommel) [reply package.json:282]: Shouldn't break

File: package.json (hunk 4 above)
Comment (mommel) [reply package.json]: Yes we can but maybe we have to remove it again

**Review by mommel — COMMENTED** [review]
All done

**Review by Illumiated — APPROVED** [review]

**Pull request merged**